functions.php

```diff
@@ -26,7 +26,7 @@ function ucf_degree_external_list_card_order( $items, $layout, $args ) {
 
 					if ( ! isset( $colleges[$college->slug] ) ) {
 						$colleges[$college->slug] = (object)array(
-							'alias'  => $college->name,
+							'alias'  => str_replace( 'College of ', '', $college->name ),
```

Comment:
Would it make sense to update the Degree CPT plugin to add college aliases to the degree results, instead of having to modify the full names here?  Or are we trying to avoid adding too much extra data to that feed?

Reply:
For the time being, I think it would be best to leave it as is. I'm concerned that adding an additional lookup on the endpoint (ends up being a nested join statement in the mysql query) is going to slow down the typeahead on the main site and online.

I think when we're able to start using the search service as the source, making the change will be less costly.